move2kube/m2k.sw.yml

```diff
@@ -177,7 +180,7 @@ states:
              refName: createNotification
              arguments:
                title: '"Move2Kube workflow " + $WORKFLOW.instanceId + " success"'
-               message: '"Move2Kube workflow  " + $WORKFLOW.instanceId + " on workspace " + .workspaceId + " and project " + .projectId + " was successful"'
-               origin: "Move2Kube Worflow"
+               message: '"Move2Kube workflow  " + $WORKFLOW.instanceId + " on workspace " + .workspaceId + " and project " + .projectId + " was successful; the output is available in the branch " + .targetBranch + " of your git repository "+ .repositoryURL"'
```

Comment:
is there an option to construct a link directly to the branch?

Reply:
unfortunately, no; the URL is not the same if you use github/bitbucket/XXX so there is no way of providing an agnostic URL
And I am not keen at having if/else